src/core/ollayerprimitive.js

```diff
@@ -10,7 +10,11 @@ goog.provide('olcs.core.OlLayerPrimitive');
  * @extends {Cesium.PrimitiveCollection}
  */
 olcs.core.OlLayerPrimitive = function(layerProjection, scene) {
-  goog.base(this);
+  if (!olcs.core.OlLayerPrimitive.initialized) {
+    olcs.core.OlLayerPrimitive.initialized = true;
+    goog.inherits(olcs.core.OlLayerPrimitive, Cesium.PrimitiveCollection);
+  }
+  Cesium.PrimitiveCollection.call(this);
```

Comment:
I added comments to the code about the direct call to the super constructor and the '`Cesium.ImageryProvider` interface'.